src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsRequestEntity.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "strike_off_objections_requests")
```

Comment:
Mongo collection name is still to be defined (checking with Steve Roberts) but it's likely to not have "_requests" at the end. Will confirm once known.

Reply:
Ok. Confirmed that both Mongo DB name and collection name will be "strike_off_objections" so "_requests" should go.

Reply:
We'll keep what we have as a placeholder for now and chnage it when we have more of a decision.

Reply:
Yep. Decision made and approved by Steve Roberts - it's "strike_off_objections".